### PR TITLE
updated storybook to include links and code

### PR DESCRIPTION
**Pull request by joetancy**
updated the docs as requested

### CODE REVIEW

**Comment by joetancy** [comment]
Have been hitting Insecure Processing of Data in one of the files, please advise! thanks!

**Comment by joetancy** [comment]
also added notification component

File: src/components/Notification.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      <div className="sgds-notification-detail">
+        {props.icon ? <span className={getIcon()}></span> : null}
+        <div className="sgds-notification-content">
+          <p className="has-text-weight-bold">{props.title}</p>
+          <p>{props.content}</p>
+        </div>
+        {props.closable ? (
+          <span
+            className="delete"
+            id="cancel-notification"
```
Comment (weijunyu) [inline src/components/Notification.js:41]: `id` attribute can be removed

**Review by weijunyu — CHANGES_REQUESTED** [review]

**Comment by weijunyu** [comment]
Only 1 fix, otherwise LGTM!

**Pull request merged**